File: Python/9thOct-8.py
def fact(number):
    if(number==0 or number==1):
        return 1
    else:
        return number*fact(number-1)
def gcd(a,b):
    if(b==0):
        return a
    else:
        return gcd(b,a%b)
n,m=map(int,input().split())
n1=fact(n)
m1=fact(m)
result=gcd(n1,m1)
print(result)               
                
            

File: JAVA/9th Oct-8.java
import java.util.*;
class GCD
{
    public long minimum(long i,long j)
    {
        return fact((i>j)?j:i);
    }
    public long fact(long n)
    {
        if(n==1)
        return 1;
        return n*fact(n-1);
    }
    public static void main (String[] args)
    {
        Scanner sc=new Scanner(System.in);
        System.out.println("Enter two numbers");
        long a=sc.nextLong();
        long b=sc.nextLong();
        GCD obj=new GCD();
        System.out.println("GCD="+ obj.minimum(a,b));
    }
}
